make cluster-sync fails with podman v4.4.0

<!-- Make sure that you visit our User Guide at https://kubevirt.io/user-guide.
-->

**What happened**:
With podman 4.4.0, make cluster-sync fails with
```bash
Waiting for rsyncd to be ready.............................failed
@ERROR: chroot failed
rsync error: error starting client-server protocol (code 5) at main.c(1859) [sender=3.2.7]
make[1]: *** [Makefile:150: cluster-build] Error 5
```

Downgrading podman to 4.2.1 fixes the issue.

**What expected to happen**
`make cluster-sync` should work with podman 4.4.0

**How to reproduce it (as minimally and precisely as possible)**:
Using [rootful podman](https://github.com/kubevirt/kubevirtci/blob/main/PODMAN.md) with version 4.4.0, just `make cluster-up` and `make cluster-sync`

**Additional context**:
As pointed out by @brianmcarey , this seems to be the recent cap change in podman. See https://github.com/containers/podman/issues/17504

--- a/hack/dockerized
+++ b/hack/dockerized
@@ -53,7 +53,7 @@ $KUBEVIRT_CRI run ${CONTAINER_ENV} -v "${BUILDER}:/root:rw${selinux_bind_options
 mkdir -p ${OUT_DIR}
 
 # Start an rsyncd instance and make sure it gets stopped after the script exits
-RSYNC_CID=$($KUBEVIRT_CRI run ${CONTAINER_ENV} -d -v "${BUILDER}:/root:rw${selinux_bind_options}" --security-opt "label=disable" --expose 873 -P ${KUBEVIRT_BUILDER_IMAGE} /usr/bin/rsync --no-detach --daemon --verbose)
+RSYNC_CID=$($KUBEVIRT_CRI run ${CONTAINER_ENV} -d -v "${BUILDER}:/root:rw${selinux_bind_options}" --security-opt "label=disable" --cap-add SYS_CHROOT --expose 873 -P ${KUBEVIRT_BUILDER_IMAGE} /usr/bin/rsync --no-detach --daemon --verbose)
 
 function finish() {
     $KUBEVIRT_CRI stop --time 1 ${RSYNC_CID} >/dev/null 2>&1

